### PR TITLE
42 - Example to show a basic form validation with vanilla JS

### PR DESCRIPTION
**Pull request by alexventuraio**
Simple form validation with JS.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -264,10 +263,14 @@ <h4>Rock Paper Scissors Game</h4>
         <img src="30DaysOfJavaScript/assets/37.png" alt="Rock Paper Scissors Game" />
       </a>
     </div>
+    <div class="item">
+      <a target="_blank" href="38 - Simple Form Validation/index.html">
+        <h4>Simple Form Validation</h4>
+        <img src="30DaysOfJavaScript/assets/38.png" alt="Simple Form Validation" />
+      </a>
+    </div>
```
Comment (alexventuraio) [inline index.html:271]: @swapnilsparsh it already is in the official web page

**Comment by swapnilsparsh** [comment]
@alexventuraio Resolve the conflicts and update the project number.

**Comment by alexventuraio** [comment]
> @alexventuraio Resolve the conflicts and update the project number.

@swapnilsparsh addressed!

**Comment by swapnilsparsh** [comment]
@alexventuraio Please resolve the conflicts.😅

**Comment by alexventuraio** [comment]
> @alexventuraio Please resolve the conflicts.😅

@swapnilsparsh  all good now! 😎

**Comment by swapnilsparsh** [comment]
@alexventuraio Thank you for your contribution. Looking forward to more such amazing contributions 🌟

**Pull request merged**